lib/methwhy.g

```diff
@@ -322,20 +319,24 @@ local flags, implied, f, extra_implications, implication, name, diff_reqs, diff_
     implied := SUB_FLAGS(implied, flags);
   fi;
 
+  reduced:= trues -> Filtered( trues,
+      i -> not ( INFO_FILTERS[i] in FNUM_TPRS
+                 and FLAG1_FILTER( FILTERS[i] ) in trues ) );
```

Comment:
For HPC-GAP, this requires `atomic readonly FILTER_REGION do ... od` around the access to `INFO_FILTERS ` and `FILTERS`, I am afraid. So (modulo typos):
```gap
reduced := functon(trues)
  atomic readonly FILTER_REGION do
    return not ( INFO_FILTERS[i] in FNUM_TPRS
                and FLAG1_FILTER( FILTERS[i] ) in trues ) );
  od;
end;
```